ci_framework/roles/use_install_yamls/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: "Create /etc/profile.d/{{ cifmw_use_install_yamls_envfile }}"
```

Comment:
we want `{{ cifmw_use_install_yamls_basedir }}` which points to `{{ cifmw_basedir }}`, with a defaults to `{{ ansible_user_dir }}/ci-framework/`.
And, to be more precise, `~/ci-framework/artifacts/{{ cifmw_use_install_yamls_envfile }}`

Adding a param to the module would be nice, like I did for the ci_make one.